frontend/src/components/TagsBar/TagsBar.js

```diff
@@ -1,26 +1,23 @@
 import map from 'lodash/map';
-import React, { Component } from 'react';
+import React from 'react';
 
 import AddTag from './AddTag/AddTag';
 import SingleTag from './SingleTag/SingleTag';
 
 import './TagsBar.css';
 
-export default class TagsBar extends Component {
-  constructor() {
-    super();
-    this.state = {
-      tags: ['VEGAN', 'HEALTHY', 'DINNER', 'CHEESY', 'KIDS', 'NUTS'],
-    };
-  }
+const TagsBar = ({ tags, deleteSelTag, showDeleteButton }) => (
+  <div className="tags-bar col-sm-10 offset-sm-1">
+    {map(tags, tag => (
+      <SingleTag
+        tag={tag.title}
+        key={tag.id}
+        deleteSelTag={deleteSelTag}
+        showDeleteButton={showDeleteButton}
+      />
+    ))}
+    <AddTag />
```

Comment:
Shouldn't the` <AddTag />` component be hidden when you're just viewing the recipe? It seems like it would only be necessary when on the create recipe form.

Reply:
We didn't work with Add Tag component. We have it on next ticket 344, where we hid it.